packages/cli/src/commands/integration/add.ts

```diff
@@ -1,116 +1,154 @@
-import { Team } from '@vercel-internals/types';
+import chalk from 'chalk';
 import open from 'open';
 import Client from '../../util/client';
-import getUser from '../../util/get-user';
+import formatTable from '../../util/format-table';
+import { packageName } from '../../util/pkg-name';
+import getScope from '../../util/get-scope';
+import list from '../../util/input/list';
+import cmd from '../../util/output/cmd';
+import indent from '../../util/output/indent';
 import { getLinkedProject } from '../../util/projects/link';
-import getTeamById from '../../util/teams/get-team-by-id';
-
-export interface IntegrationProduct {
-  id: string;
-  slug: string;
-  name: string;
-  shortDescription: string;
-  type: 'storage' | string;
-}
-
-export interface Integration {
-  id: string;
-  slug: string;
-  name: string;
-  products?: IntegrationProduct[];
-}
+import {
+  BillingPlan,
+  Integration,
+  IntegrationInstallation,
+  IntegrationProduct,
+  Metadata,
+} from './types';
+import { createMetadataWizard, MetadataWizard } from './wizard';
+import {
+  fetchIntegration,
+  fetchInstallations,
+  fetchBillingPlans,
+  provisionStoreResource,
+  connectStoreToProject,
+} from './client';
 
 export async function add(client: Client, args: string[]) {
   if (args.length > 1) {
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (blocking):** Check args more carefully

This check prevents invocations like:

```
vc i supabase --cwd=some/path
```

Reply:
Very good catch. Should be fixed now.